src/Items/ItemSlab.h

```diff
@@ -63,11 +69,28 @@ class cItemSlabHandler:
 				((ClickedBlockMeta & 0x08) != 0)
 			)
 			{
+				a_World.BroadcastSoundEffect(PlaceSound, a_BlockX + 0.5, a_BlockY + 0.5, a_BlockZ + 0.5, Volume, Pitch);
 				return a_Player.PlaceBlock(a_BlockX, a_BlockY, a_BlockZ, m_DoubleSlabBlockType, ClickedBlockMeta & 0x07);
 			}
 		}
 
+		// Checking the type of block that should be placed 
```

Comment:
Theres one extra space one the end of this line which is being flagged by the build server.